contracts/contracts/Utility.sol

```diff
@@ -0,0 +1,74 @@
+// SPDX-License-Identifier: MIT
```

Comment:
To keep the repo a bit more organized, should we move utilities contract like this one to their own folder named for instance utilities/helpers?

Reply:
sounds good!